pcstac/tests/conftest.py

```diff
@@ -43,75 +17,99 @@ def event_loop():
 
 
 @pytest.fixture(scope="session")
-async def pqe_pg():
-    print(f"Connecting to write database {settings.reader_connection_string}")
-    print("writer conn string", settings.reader_connection_string)
-    conn = await asyncpg.connect(dsn=settings.reader_connection_string)
-    val = await conn.fetchval("SELECT true;")
-    print(val)
-    await conn.close()
-    version = await pypgstac.run_migration(dsn=settings.reader_connection_string)
-    print(f"PGStac Migrated to {version}")
-
-    yield settings.reader_connection_string
-    await conn.close()
-
-
-@pytest.fixture(scope="session")
-def api_client(pqe_pg):
-    print("creating client with settings", settings, settings.reader_connection_string)
-    extensions = [
-        QueryExtension(),
-        SortExtension(),
-        FilterExtension(),
-        FieldsExtension(),
-        TokenPaginationExtension(),
-        ContextExtension(),
-    ]
-    search_get_request_model = create_get_request_model(extensions)
-    search_post_request_model = create_post_request_model(
-        extensions, base_model=PCSearch
-    )
-    api = PCStacApi(
-        title="test title",
-        description="test description",
-        api_version="1.0.0",
-        settings=Settings(debug=True),
-        client=PCClient.create(
-            post_request_model=search_post_request_model,
-        ),
-        extensions=extensions,
-        app=FastAPI(default_response_class=ORJSONResponse),
-        search_get_request_model=search_get_request_model,
-        search_post_request_model=search_post_request_model,
-    )
-
-    return api
-
-
-@pytest.mark.asyncio
-@pytest.fixture(scope="session")
-async def app(api_client):
-    time.time()
-    app = api_client.app
-    await connect_to_db(app)
-
-    yield app
-
-    await close_db_connection(app)
-
-
-@pytest.mark.asyncio
-@pytest.fixture(scope="session")
-async def app_client(app):
-    async with AsyncClient(app=app, base_url="http://test") as c:
-        yield c
-
-
-@pytest.fixture
-def load_test_data() -> Callable[[str], Dict]:
-    def load_file(filename: str) -> Dict:
-        with open(os.path.join(DATA_DIR, filename)) as file:
-            return json.load(file)
-
-    return load_file
+async def app_client():
```

Comment:
MAJOR CHANGE! 

I think it's safer to not update/change the default database. Here I'm creating a new db `pgstactestdb` and running pgstac migration + ingest of test data. 

At the end of the tests, the database will be teared down.